CONFIGURATION.md

```diff
@@ -0,0 +1,122 @@
+# Configure your GitHub Copilot Extension
+
+## Create a GitHub app
+
+Head to your settings page on GitHub, and at the bottom, to the left, click on the **Developer Settings** link. This will bring you to your **GitHub apps**. (You can also directly navigate to your **GitHub apps** page at https://github.com/settings/apps.) Configure the following:
+
+### General Settings
+
+1. Enter a **GitHub App name**, e.g. my-copilot-extension
+2. Enter a URL for the homepage. This can be the same as the test URL for now.
+3. Set the **Callback URL** (currently required). This can be the same as the test URL for now. 
+<!-- ***** If these (Homepage URL and Callback URL) are indicated in the first part of the article, how best to reference it here? ***** --> 
```

Comment:
You can use the screen shot from the blog post. In fact you can save all the images from the relevant sections.

Reply:
Can I make a directory for the screenshots? Something like `/docs/images`?

Reply:
Whatever you think makes sense. That sounds good to me. If we create a docs folder, consider putting the markdown file in there in the root as well.